lasy/profiles/combined_profile.py

```diff
@@ -32,23 +32,59 @@ class CombinedLongitudinalTransverseProfile(Profile):
         :math:`p_y` is the second element of the list. Using complex
         numbers enables elliptical polarizations.
 
-    laser_energy : float (in Joule)
-        The total energy of the laser pulse. The amplitude of the laser
-        field (:math:`E_0` in the above formula) is automatically
-        calculated so that the pulse has the prescribed energy.
-
     long_profile : :class:`.LongitudinalProfile`
         Defines the longitudinal envelope of the laser, i.e. the
         function :math:`\mathcal{L}(t)` in the above formula.
 
     transverse_profile : :class:`.TransverseProfile`
         Defines the transverse envelope of the laser, i.e. the
         function :math:`\mathcal{T}(x, y)` in the above formula.
+
+    laser_energy : float (in Joule)
+        The total energy of the laser pulse. The amplitude of the laser
+        field (:math:`E_0` in the above formula) is automatically
+        calculated so that the pulse has the prescribed energy.
+        Only used in case where laser is neither a plane wave or a continuous wave laser
+
+    peak_fluence : float (in J/m^2)
+        The peak fluence of the laser pulse. The amplitude of the laser
+        field (:math:`E_0` in the above formula) is automatically
+        calculated so that the pulse has the prescribed peak fluence.
+        Only used in case where laser is a continuous wave laser
```

Comment:
I think that is a good option in general and should be usable.
```suggestion
        Required for a CW (monochromatic) profile.
```